SpatialGDK/Extras/schema/server_worker.schema

```diff
@@ -7,7 +7,7 @@ import "unreal/gdk/spawner.schema";
 type ForwardSpawnPlayerRequest {
     SpawnPlayerRequest spawn_player_request = 1;
     UnrealObjectRef player_start = 2;
-    string client_worker_id = 3;
+    EntityId client_system_entity_id = 3;
```

Comment:
Wouldn't this already be contained in `SpawnPlayerRequest`? (field 1)

Reply:
Yes good point